install.sh

```diff
@@ -1,45 +1,135 @@
 #!/bin/bash
 
-NET_SUBNET="192.168.231"
+
+# Copy the configuration file
+COPY='rsync -uv ' # Just because some platforms don't know
```

Comment:
Then, by it error'ing out, dev will fulfil handle this. Sadly BSD does not implement `cp --update` option.